ui.apps/src/main/content/jcr_root/apps/core/cif/components/content/teaser/v1/teaser/clientlib/editor/js/teaser.js

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
```

Comment:
Please add a test for this file and make sure that https://github.com/adobe/aem-core-cif-components/blob/master/ui.apps/karma.conf.js includes it for coverage.